protocol/x/accountplus/keeper/authenticators.go

```diff
@@ -106,3 +112,138 @@ func (k Keeper) SetNextAuthenticatorId(ctx sdk.Context, authenticatorId uint64)
 	store := ctx.KVStore(k.storeKey)
 	store.Set([]byte(types.AuthenticatorIdKeyPrefix), b)
 }
+
+// GetAuthenticatorExtension unpacks the extension for the transaction, this is used with transactions specify
+// an authenticator to use
+func (k Keeper) GetAuthenticatorExtension(exts []*codectypes.Any) types.AuthenticatorTxOptions {
+	for _, ext := range exts {
+		var authExtension types.AuthenticatorTxOptions
+		err := k.cdc.UnpackAny(ext, &authExtension)
```

Comment:
Does this need to be a keeper method? It just takes in the codec

Reply:
makes sense. I actually moved this to the other PR https://dydx-team.slack.com/archives/C03FP255RRU/p1727122423450509 since it's being used there and not here